### ISSUE TITLE
New version of Bun 1.1.5 dont works with stripe

### ISSUE THREAD

**Issue by irvile** [oven-sh/bun#10585]
### What version of Bun is running?

1.1.5+b257a3097

### What platform is your computer?

Darwin 23.1.0 arm64 arm

### What steps can reproduce the bug?

```js
import Stripe from 'stripe'

const stripe = new Stripe(stripePrivateKey)
const customers = await stripe.customers.list({ limit: 100 })
console.log('my customers')
```

### What is the expected behavior?

This case, see the log 'my customers'.

### Additional information

Hi, after upgrade to new version my service with Stripe stop to work.

This version 1.1.4+fbe2fe0c3 works fine.

but the new one 1.1.5+b257a3097 dont.

**Comment by mhslr** [comment]
Same story here.
My service is deployed on Google Cloud Run.
When I upgraded from v1.1.4 to v1.1.5, the following lines stopped working, and instead caused timeouts (after 180 seconds).

```ts
    const session = await stripe.billingPortal.sessions.create({
      customer: user.stripeCustomerId,
    });
```

**Comment by abaudhuin** [comment]
Seems to be fixed in 1.1.6. Do you confirm ?

**Comment by Jarred-Sumner** [comment]
This is fixed in Bun v1.1.6
